profiles/source/Source_HostSettings.yaml

```diff
@@ -0,0 +1,36 @@
+Device:
```

Comment:
format is not correct, please use yaml linter.

There is no such thing as a combined setting.

Your either a dsAudio or a dsHdmiIn or 

your not all these at all times.


file: dsAudioConfiguration.yaml

```yaml
dsAudio:
 type: sink
 name: element
  features:
    extendedEnumsSupported: false
```